### ISSUE TITLE
[Feature Request] Disable "always on top" behavior for quake terminal

### ISSUE THREAD

**Issue by alraban** [diegodario88/quake-terminal#8]
I recently migrated from the old "quake-mode" extension that no longer works in Gnome 45.  Thanks so much for making a replacement that works in Gnome 45!  

However, the older extension would automatically hide the quake terminal when you clicked on a window that's "behind" it.  For example if the quake terminal occupied only the top half of the screen, clicking a window in the lower half of the screen would bring that window to the "top" and hide the quake terminal (like it was a normal desktop window), which was nice behavior as I don't typically expect a window to exhibit "always on top" behavior unless it requires immediate attention.  

This extension instead makes itself "always on top" until you press the launch key again.  As a related issue, if you click outside the quake terminal not only does it not hide itself, pressing the launch key once won't hide the terminal, instead just refocusing the terminal which is counterintuitive (i.e., if you've clicked away you have to press the launch key twice to hide the terminal, once to focus the terminal and once to hide it).  

My requests: 

1. Please consider making the window behave like other windows where if you click away from the quake terminal or click a  window that's underneath it it will allow itself to be backgrounded (or at least provide an option to disable always on top behavior).
2.  If possible, a single press of the launch key or key chord should show the terminal if hidden or hide the terminal if shown.  It's counter intuitive to have to press the launch key twice to hide a visible quake terminal just because it's not currently in focus.

If you've used the old "quake-mode" extension these are the default behaviors there so you might already be familiar with what I mean.

**Comment by diegodario88** [comment]
@alraban, Thank you for your valuable feedback, and I am glad to hear that you found our replacement extension useful in Gnome 45! I appreciate your input on the behavior of the quake terminal, and I've made improvements to address your concerns.

Now, there's a switch option in the preferences called `Auto Hide.` When enabled, this allows the terminal window to automatically hide as soon as it loses focus.

Additionally, when that option is `disabled`, the counterintuitive behavior that required the user to press the launch key twice, once to bring the focus and another to hide, has been fixed.
